File: api/code-samples/javascript/klaytnDebug/blockchainInspection/getBlockRlp.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const blockNumber = "200"
    
    sdk.debug.getBlockRlp(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/blockchainInspection/getBlockRlp.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockHash = "latest"

sdk = OpenSDK(host)
debug_response = sdk.debug.get_block_rlp(blockHash)

print(json.loads(debug_response.response.data))
